src/main/java/decodes/xml/RoutingSpecParser.java

```diff
@@ -216,51 +217,47 @@ public void ignorableWhitespace( char[] ch, int start, int length ) throws SAXEx
 	 */
 	public void set( int tag, String str ) throws SAXException
 	{
-		switch(tag)
+		try
 		{
-		case outputFormatTag:
-			routingSpec.outputFormat = str;
-			break;
-		case outputTimeZoneAbbrTag:
-			routingSpec.outputTimeZoneAbbr = str;
-			break;
-		case presentationGroupNameTag:
-			routingSpec.presentationGroupName = str;
-			break;
-		case sinceTimeTag:
-			routingSpec.sinceTime = str;
-			break;
-		case untilTimeTag:
-			routingSpec.untilTime = str;
-			break;
-		case consumerTypeTag:
-			routingSpec.consumerType = str;
-			break;
-		case consumerArgTag:
-			routingSpec.consumerArg = str;
-			break;
-		case propertyTag:
-			if (propName == null)
-				throw new SAXException("Property value without name!");
-			routingSpec.getProperties().setProperty(propName, str);
-			propName = null;
-			break;
-		case lastModifyTimeTag:
-/*
-MJM 20031023 - Don't use the LMT contained in XML file. It may not agree
-with the File.lastModified() calls used elsewhere.
-			try
-			{
-				routingSpec.lastModifyTime = 
-					Constants.defaultDateFormat.parse(str);
-			}
-			catch(Exception e)
+			final String realValue = Property.getRealPropertyValue(str, str);
```

Comment:
I will likely revert this back to the original. Thinking about it more, property values should be expanded at time of use... at least the date based ones at any rate. Going to have to think about it a bit longer.

Alternatively both is valid since we can implement a `\$ to escape the processing. Though that's still a breaking change.